Change some sample disapprove comments to feel less 'stern'. (#1264)
exercism/website-copy: automated-comments/javascript/gigasecond/dont_use_date_parse.md

> Your solution used `Date.parse(%<parameter.name>)`. This works, but isn't _intended_ to be used with a `Date` object. In fact, in order to work it first coerces the input value to a `string` and then parses it back into a date, which is unnecessary in this case.

Reviewer: ```suggestion
Passing a `Date` as input to `Date.parse(%<parameter.name>)` works, but this function isn't _intended_ to work with `Date` objects. In fact, when `Date.parse` receives a `Date` parameter, it first coerces it to a `string` and then parses it back into a `Date`, which is unnecessary in this case.
```

> It would be better to use `%<parameter.name>.getTime()` as [`Date#getTime`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getTime) doesn't coerce the input but works on the Date directly.

Reviewer: I'm not a fan of "It would be better". Maybe we rephrase this a bit?

Reviewer 2: "This works, but it's more efficient to use... because..."

> It would be better to use `%<parameter.name>.getTime()` as [`Date#getTime`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getTime) doesn't coerce the input but works on the Date directly.

Reviewer 3: ```suggestion
It is better to use `%<parameter.name>.getTime()` as [`Date#getTime`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getTime) doesn't coerce the input but works on the Date directly.
```

Reviewer 3: Gathered from @kotp suggestion.

Author: I prefer "would be" I think. "is" feels a lot stronger, "would be" feels a bit more chatty and friendly to me.

Reviewer 4: I use "would be" in conjunction with "if something is done".  It "is better" because of a specific reason, as given here.
